### ISSUE TITLE
(Interactive Description) - Update go-beyond-edge response for mass controls

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#293]
In https://github.com/phetsims/gravity-force-lab-basics/issues/291#issuecomment-888333627, we noted an issue for the change mass number spinner interactions.

In interactive description we have go-beyond-edge responses that deliver mainly qualitative current state information when a learner tries to go beyond a min or max value.

The go-beyond-edge responses for the position/distance **mass sphere sliders are working fine.** They are:
When Force Values checked:
- {{Far from}} mass {{2}}, force arrows {{tiny}}, forces {{10.9}} newtons.

When Force Values NOT checked:
- {{Far from}} mass {{2}}, force arrows {{tiny}}. 

For the number spinners, we have something similar, but we are still getting the force value in newtons when Force Values is not checked. @jessegreenberg, please remove the newton value when Force Values is not checked, the mass responses would look like this:

When Force Values checked:
- {{Much much smaller than}} mass {{2}}, force arrows {{tiny}}, forces {{16.7}} newtons.

When Force Values NOT checked:
- {{Much much smaller than}} mass {{2}}, force arrows {{tiny}}.

**Comment by jessegreenberg** [comment]
Here is the function that generates this string, and it is currently only used in this case so seems safe to change:
```js
  /**
   * @public
   * @param {ISLCObjectEnum} thisObjectEnum
   * @returns {string}
   */
  getMassMaxMinText( thisObjectEnum ) {
    return StringUtils.fillIn( massMaxMinBorderTextString, {
      relativeSize: this.getRelativeSizeOrDensity( thisObjectEnum, true ),
      otherObjectLabel: this.getOtherObjectLabelFromEnum( thisObjectEnum ),
      forceVectorSize: this.forceDescriber.getForceVectorsSize(),
      force: this.forceDescriber.getFormattedForce(),
      unit: this.forceDescriber.units
    } );
  }
```

We should be able to remove force and units when not force values are not checked.

**Comment by jessegreenberg** [comment]
Done in the above commit, will prepare a version to review shortly.

**Comment by jessegreenberg** [comment]
@terracoda can you please review in this version? https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.1.0-dev.36/phet/gravity-force-lab-basics_en_phet.html

**Comment by terracoda** [comment]
This sounds great. Thanks @jessegreenberg.
